src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -386,6 +397,8 @@ private void getChanges2(ChangeEventSourceContext context,
                                 table, streamId, tabletId,
                                 cp.getTerm(), cp.getIndex(), cp.getKey(), cp.getWrite_id(), cp.getTime(), schemaStreamed.get(tabletId));
 
+                        LOGGER.info("Processing {} records from getChanges call",
```

Comment:
This log should also be at the `debug` level otherwise they will clog up the whole log file.